iso/boot/ubuntu/grub.cfg

```diff
@@ -0,0 +1,21 @@
+if loadfont unicode ; then
```

Comment:
what is the purpose of this file? can you add a comment explaining it and official documentation if any? otherwise it would be very difficult to get context for someone else who want to work on this

Reply:
Also is this only ubuntu specific?

Reply:
The purpose is  configuring the boot. It's actually pretty similar to the Debian one we already have here: https://github.com/dappnode/DAppNode/blob/master/iso/boot/grub.cfg

Reply:
It just inserts some needed modules, sets the theme (with the Dappnode logo and so on) and allows only 1 option in the menu, which is installing Dappnode